README.md

```diff
@@ -1,3 +1,108 @@
-# Spark Fat Jar Template
+# Setting up a Scheduled Spark Scala job on AWS
 
-Read me will contain article in future. 
\ No newline at end of file
+Have you written a Scala Spark job that processes a massive amount of data on a stupid amount of RAM and you want to figure out a way to run it daily/weekly/monthly on a schedule on [AWS](https://aws.amazon.com)?
+Well, I did and I could not find a good tutorial on the full process to get the spark job running, so here is mine.
+Included in this article and accompanying repository should be everything you need to take your Scala Spark job and get it running on AWS [Data Pipeline](https://aws.amazon.com/datapipeline/) and [EMR](https://aws.amazon.com/emr/).
+
+## Code Repo
+
+This tutorial is not going to walk you through the process of actually writing your specific Scala Spark job to do whatever number crunching you need. 
+There are [plenty of](https://www.analyticsvidhya.com/blog/2017/01/scala/) [resources](https://spark.apache.org/docs/latest/quick-start.html) [available](https://www.coursera.org/learn/scala-spark-big-data) already to get you started on that. 
```

Comment:
This might be your intended style, but you could also put these resources into a numbered set "plenty of resources available [1](...) [2](...) [3](...)" to make it clear that they are different links.